frontend/src/components/signup.tsx

```diff
@@ -1,58 +1,53 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Eslint quick fixes, isValidLength & hasNumber & hasSpecialChar are unused

Reply:
not sure what you're trying to say here but you know what I'm gonna say hahaha